frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -1,17 +1,43 @@
 <script setup lang="ts">
+import { computed } from 'vue';
 import LabelText from '@/components/design-system/typography/LabelText.vue';
 
 export interface Props {
   active?: boolean;
 }
 
-withDefaults(defineProps<Props>(), {
+const classes = computed(() => ({
+  'pill--active': props.active,
+}));
+
+const props = withDefaults(defineProps<Props>(), {
   active: false,
 });
 </script>
 
 <template>
-  <LabelText>
-    <slot />
-  </LabelText>
+  <div :class="['pill', classes]">
+    <LabelText>
+      <slot />
+    </LabelText>
+  </div>
 </template>
+
+<style lang="scss" scoped>
+.pill {
+  border-color: rgba(var(--color-neutral-border), 1);
```

Comment:
Pokud  je alpha na 1, není nutné používat `rgba`, stačí `rgb`.

Reply:
Jj, z nějakého důvodu mi to nedošlo... Je to upravené.